inc/class-blocks-animation.php

```diff
@@ -63,74 +75,72 @@ public function enqueue_editor_assets() {
 		wp_script_add_data( 'otter-count', 'defer', true );
 	}
 
+
+
 	/**
-	 * Load Gutenberg assets.
+	 * Load assets in frontend.
 	 *
-	 * @since   1.0.0
-	 * @access  public
+	 * @param string $block_content Content of block.
+	 * @param array  $block Block Attributes.
+	 * @return string
+	 * @since 2.0.5
 	 */
-	public function enqueue_block_frontend_assets() {
+	public function frontend_load( $block_content, $block ) {
+
 		if ( function_exists( 'is_amp_endpoint' ) && is_amp_endpoint() ) {
-			return;
+			return $block_content;
 		}
 
-		global $post;
-
-		if ( is_singular() && strpos( get_the_content( null, false, $post ), '<!-- wp:' ) === false ) {
-			return;
+		if( strpos( $block_content, 'animated' ) && ! self::$scripts_loaded['animation'] ) {
```

Comment:
I would add the flag first and after the strpos since the condition, in this way the strpos is no longer executed once the flag is added.